packages/cubejs-testing/birdbox-fixtures/driver-test-data/CAST.js

```diff
@@ -29,6 +30,13 @@ export const DATE_CAST = {
     DATE_PREFIX: 'parse_date(\'%Y-%m-%d\', ',
     DATE_SUFFIX: ')',
   },
+  // The default range queries for `refreshKey: every '...'` use `DATETIME(timestamp, tz)`.
+  // parse_date returns a DATE instead of a TIMESTAMP, and refresh queries fail with
+  // "Error: Could not cast literal "UTC" to type TIME"
+  bigquery2: {
+    DATE_PREFIX: 'parse_timestamp(\'%Y-%m-%d\', ',
```

Comment:
I've checked the `parse_timestamp` casting with the existing BigQuery driver tests, and it looks fine. Can you replace the current way with yours and keep this object consistent?